__tests__/index.test.js

```diff
@@ -183,3 +183,46 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("PATCH 200: returns updated article", () => {
+    const votesToBeUpdatedBy = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(votesToBeUpdatedBy)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.updatedArticle).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 101,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("PATCH 404: returns correct error message if passed article that doesn't exist", () => {
+    const votesToBeUpdatedBy = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/9999")
+      .send(votesToBeUpdatedBy)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Article Not Found");
+      });
+  });
```

Comment:
Really good error handling test for the article id being valid but none existent, but dont forget to include an article id that is invalid i.e. /api/articles/banana